ui/imodel-components-react/src/imodel-components-react/timeline/SolarTimeline.tsx

```diff
@@ -700,39 +707,37 @@ export class SolarTimeline extends React.PureComponent<
             formatTime={this._formatTime}
             isPlaying={this.state.isPlaying}
           />
-          <div className="speed-container">
-            <span title={this._speedLabel}>{speed}x</span>
-            <SpeedTimeline
-              className="speed"
-              onChange={this._onSpeedChange}
-              speed={this.state.speed}
-            />
-          </div>
-          {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events */}
-          <span
-            title={this._loopLabel}
-            className={classnames(
-              "icon",
-              !loop && "no-loop-playback",
-              loop && "loop-playback"
-            )}
+
+          <select className="xyz" name="speed" id="speed">
```

Comment:
I like this approach, but why it's not an iTwinUI component? Do we need/use name&id attributes? Rename class name.

Reply:
Btw we can remove `SpeedTimeline` component since it is internal and not used anywhere else.

Reply:
@mayank99 and I are discussing this.  iTwinUI does not offer `<select>` styling atm so I made something custom just to try it out for now.

Reply:
Why isn't https://itwinui.bentley.com/docs/dropdownmenu an option?

Reply:
[PR created to add native `<select>` element styling to iTwinUI](https://github.com/iTwin/iTwinUI/pull/1879), a PR will follow this adding borderless option.

Reply:
While the above PR nears completion, I've created [a follow-up PR to add borderless variant to select in iTwinUI](https://github.com/iTwin/iTwinUI/pull/1886).  Once these 2 PRs merge, we can use that in this PR.

Reply:
I think we can complete this PR, while we wait for the borderless variant.